cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -136,7 +136,7 @@ install_docker(){
 			repo_url="https://download.docker.com/linux/centos/docker-ce.repo"
 			sudo yum-config-manager --add-repo "$repo_url"
 			# With this we will be able to install docker 18.06
-			if [ "$ID" == "centos" ] && [ "$VERSION_ID" -ge "8" ]; then
+			if [ "$ID" == "centos" ] && [ "$VERSION_ID" -ge "8" ] || [ "$ID" == "rhel" ] && [ "$VERSION_ID" -ge "8" ]; then
```

Comment:
@GabyCT, this check is too complicated for my simple mind.
I'd suggest we do:
- Remoe the check for the "$ID", as at this point it can only be "centos" or "rhel"
- Check only for the "$VERSION_ID", as we'd do the change for both "centos" or "rhel", if those are >= 8

So, in the end the check would be:
```
if [ "$VERSION_ID" -ge "8" ]; then
  ...
fi
```

Reply:
I agree this should be simplified but is `VERSION_ID` guaranteed to be an integer for any release of CentOS/RHEL? The spec says it could be a floating point or contain alphabetics:

- https://www.freedesktop.org/software/systemd/man/os-release.html

Reply:
@jodh-intel, you're right about your point, but I sincerely think we can simplify it even more, as we can simply do the `sed -i ...` regardless of the version.

Do we test if RHEL / CentOS 7? I don't think so. But if we do, changing $releasever to 7 would just match;
For 8, we want to do that for both CenOS and RHEL.

Reply:
@fidencio, in the past we tested RHEL 7 and CentOS 7.

Reply:
Okay, so, please, just remove the conditions and do the sed unconditionally.